.github/workflows/fe-pr-snapshot.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         run: npm ci
 
       - name: Build and Push to Jfrog NPM Registry
-        uses: collaborationFactory/github-actions/.github/actions/artifacts@release/25.1
+        uses: collaborationFactory/github-actions/.github/actions/artifacts@fix/PFM-ISSUE-28573-Fix-usage-of-npm-unpublish-in-delete-snapshot
```

Comment:
Make sure this is changed back before merging

Reply:
done